scripts/install.ps1

```
@@ -74,79 +55,41 @@ function Show-Banner {
     }
-
+    $arch = if ($env:PROCESSOR_ARCHITECTURE -eq "ARM64") { "arm64" } else { "amd64" }
     Write-Success "Platform: Windows ($arch)"
-    return $arch
 }
 
-# ============================================================================
-# Prerequisites
-# ============================================================================
-
 function Test-Prerequisites {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Test-Prerequisites' uses a plural noun. A singular noun should be used instead.

Suggested fix: Singularized correction of 'Test-Prerequisites'

(PSUseSingularNouns)